File: app/src/main/java/com/nytimes/android/external/register/GithubApi.java
package com.nytimes.android.external.register;

import com.nytimes.android.external.register.model.Repository;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface GithubApi {
    @GET(BuildConfig.GITHUB_ENDPOINT)
    Observable<Repository> getPlayBillingRepository();
}

File: app/src/main/java/com/nytimes/android/external/register/GithubApi.kt
package com.nytimes.android.external.register

import com.nytimes.android.external.register.model.Repository

import io.reactivex.Observable
import retrofit2.http.GET

interface GithubApi {
    @get:GET(BuildConfig.GITHUB_ENDPOINT)
    val playBillingRepository: Observable<Repository>
}
